controllers/mdutil/util.go

```
@@ -701,7 +701,7 @@ func DeepHashObject(hasher hash.Hash, objectToWrite interface{}) {
 		DisableMethods: true,
 		SpewKeys:       true,
 	}
-	printer.Fprintf(hasher, "%#v", objectToWrite)
```

why are we ignoring this error?

---

Good question, we probably shouldn't

---

Implemented err handling now